Resources/Prototypes/Entities/Mobs/NPCs/pets.yml

```diff
@@ -601,6 +601,9 @@
   id: MobSpiderShiva
   description: The first defender of the station.
   components:
+  - type: Prying
+    pryPowered: false
+    force: false
   - type: InteractionPopup
```

Comment:
No other entity has this so this looks like a bandaid, are you sure it shouldn't just be removed entirely.